.jshintrc

```diff
@@ -10,7 +10,7 @@
     "latedef": "nofunc",
     "laxbreak": true,
     "maxdepth": 4,
-    "maxparams": 5,
+    "maxparams": 6,
```

Comment:
Instead of increasing allowed maxparams for the entire codebase, allow it at the single call site that violates the rule by disabling/enabling the specific jshint rule around the code in violation. See http://jshint.com/docs/ for specific enable/disable syntax.